float64 center_beam